Makefile

```diff
@@ -104,7 +104,7 @@ endif
 out/minikube-windows-amd64.exe: out/minikube-windows-amd64
 	cp out/minikube-windows-amd64 out/minikube-windows-amd64.exe
 
-out/minikube-%-amd64: pkg/minikube/assets/assets.go $(shell $(MINIKUBEFILES))
+out/minikube-%-$(GOARCH): pkg/minikube/assets/assets.go $(shell $(MINIKUBEFILES))
```

Comment:
This has already been changed (in c7037c434be04defcaf4d216cc6d10d66263f078). But it doesn't really help, anyway.